build-config/resources/Ballerina.toml

```
@@ -8,3 +8,9 @@ repository = "https://github.com/ballerina-platform/module-ballerinax-aws.redshi
 icon = "icon.png"
 license = ["Apache-2.0"]
 distribution = "2201.10.2"
+
+[[platform.java17.dependency]]
+groupId = "io.ballerina.lib"
+artifactId = "aws.redshiftdata-native"
+version = "0.1.0-SNAPSHOT"
+path = "../native/build/libs/aws.redshiftdata-native-0.1.0-SNAPSHOT.jar"
```

This should be a placeholder, as we replace the version with the actual value at the build.

```suggestion
version = "@project.version@" 
path = "../native/build/libs/aws.redshiftdata-native-@project.version@.jar"
```

---

Fixed with [ceaee0a97465bdcaf7b55989cd3dc7749c4b06a2](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/pull/3/commits/ceaee0a97465bdcaf7b55989cd3dc7749c4b06a2)